plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -99,6 +100,17 @@ public CubridPlanNode getParent() {
     public Collection<CubridPlanNode> getNested() {
         return nested;
     }
+    
+    @Nullable
+    @Override
+    public Color getNodeColor() {
+        if (name.equals("iscan")) {
+            return new Color(0, 255, 0);
+        } else if (name.equals("sscan")) {
+            return new Color(255, 0, 0);
+        }
```

Comment:
The `Color` class is backed by a native OS resource, so it must be disposed of after use.

Moreover, the colors chosen might (and will) look bad against the dark theme, so it's always better to make them customizable through the `org.eclipse.ui.themes` extension point whenever possible.